hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/webapp/TestFederationInterceptorREST.java

```diff
@@ -156,6 +172,31 @@ public void setUp() {
       Assert.fail();
```

Comment:
This is from before but if we just surface the exception we don't need to add the fail().

Reply:
I will fix it.